edx_proctoring/templates/proctoring/seq_proctored_exam_instructions.html

```diff
@@ -0,0 +1,125 @@
+      {% endblocktrans %}
+    </p>
+  </div>
+</div>
+<div class="footer-sequence border-b-0 padding-b-0">
+  <span> {% trans "Note: Once you complete installation and set up, your timed exam will begin." %} </span>
+    <p>
+    {% blocktrans %}
+      Please be prepared to start the exam and follow all of the guidelines of an {{platform_name}} proctored exam.
+    {% endblocktrans %}
+  </p>
+  <p class="proctored-exam-option">
+    {% blocktrans %}
+      Don't want to take this as a proctored exam? <a href="#">Take this as an open exam instead.</a>
+    {% endblocktrans %}
+  </p>
+</div>
+{% include 'proctoring/seq_proctored_exam_footer.html' %}
+
+<script type="text/javascript">
+
+  var _waiting_for_proctored_interval = null;
+
+  $(document).ready(function(){
+
+    var hasFlash = false;
+    try {
+      var fo = new ActiveXObject('ShockwaveFlash.ShockwaveFlash');
+      if (fo) {
+        hasFlash = true;
+      }
+    } catch (e) {
+      if (navigator.mimeTypes
+            && navigator.mimeTypes['application/x-shockwave-flash'] != undefined
+            && navigator.mimeTypes['application/x-shockwave-flash'].enabledPlugin) {
+        hasFlash = true;
+      }
+    }
+
+    if (hasFlash) {
+      $('.copy-to-clipboard').html(
+        '<object classid="clsid:d27cdb6e-ae6d-11cf-96b8-444553540000" \
+        width="110" \
+        height="14" \
+        id="clippy" > \
+          <param name="movie" value="/static/proctoring/clippy.swf"/> \
+          <param name="allowScriptAccess" value="always" /> \
+          <param name="quality" value="high" /> \
+          <param name="scale" value="noscale" /> \
+          <param NAME="FlashVars" value="text={{exam_code}}"> \
+          <param name="bgcolor" value="#F2F4F5"> \
+          <embed src="/static/proctoring/clippy.swf" \
+            width="110" \
+             height="14" \
+             name="clippy" \
+             quality="high" \
+            allowScriptAccess="always" \
+            type="application/x-shockwave-flash" \
+            pluginspage="http://www.macromedia.com/go/getflashplayer" \
+            FlashVars="text={{exam_code}}" \
+            bgcolor="#F2F4F5" \
+          /> \
+        </object>'
+      );
+    }
+
+    _waiting_for_proctored_interval = setInterval(
+      poll_exam_started,
+      5000
+    );
+  });
+
+  function poll_exam_started() {
+    var url = $('.instructions').data('exam-started-poll-url')
+    $.ajax(url).success(function(data){
+      if (data.status === 'ready_to_start') {
+        if (_waiting_for_proctored_interval != null) {
+          clearInterval(_waiting_for_proctored_interval)
+        }
+        // Let the student know exam is ready to start.
+        // This alert may or may not bring the browser window back to the
+        // foreground (depending on browser as well as user settings)
+        alert('{% trans "Your proctored exam has started, please click OK to enter into your exam." %}')
```

Comment:
"Your proctoring session has started. Click **OK** to start taking your exam."
@chrisndodge Is this accurate?